### DISCUSSION TITLE
images not loading

### DISCUSSION THREAD

**Discussion by arun-arunisto** [community/community#33253] [Other Feature Feedback, Questions, & Ideas]
images missing or cracked that i uploaded on my repository weeks ago. and these images that i used on my client websites now its not loading anywhere!

**Comment by airtower-luna** [comment]
If you want help make sure to include a link to your repository and site where you see the problem, so people can actually take a look! :wink:

**Comment by jfullstackdev** [comment]
you might not notice that the repo is public, that's a possibility
